### ISSUE TITLE
save

### ISSUE THREAD

**Issue by peterdudfield** [openclimatefix/neso-solar-consumer#5]
Svaing the forecast can be done using https://github.com/openclimatefix/nowcasting_datamodel/blob/main/nowcasting_datamodel/save/save.py#L21

**Comment by siddharth7113** [comment]
"Hi @peterdudfield, for saving forecasts using the save() function, would it be sufficient to provide the forecasts and session parameters? Since flags like update_national, apply_adjuster, and save_to_last_seven_days are optional, I assume they are not needed for this particular case. Please let me know if any additional parameters are critical. Thanks!"

**Comment by siddharth7113** [comment]
I will start working on this, once this is done , I  will shift to update the tests for all 3 fetch, format and save function for issue #27
